# States for each of left light and right light. TRUE represents on; FALSE represents off.

bool left
bool right